pyrobosim/pyrobosim/navigation/path_planner.py

```diff
@@ -34,8 +34,7 @@ def __init__(self, planner_type, **planner_config):
             )
             return None
         if not planner_config:
-            warnings.warn(f"No planner configuration provided", UserWarning)
-            return None
+            planner_config = {}
```

Comment:
@ibrahiminfinite with this change you should be able to create a path planner without actually passing in a dictionary, and it should use default arguments. I don't see why this shouldn't be supported, but tagging you in case I missed something.

Reply:
Planners need either a `World` or `OccupancyGrid` to actually do the planning (for collision checking), those are passed in via the config object.

Reply:
Ah, yes, maybe I'll reinstate the warning and say something more like your comment to tell users what they may be missing. Thanks!